src/lib/bootstrap_controller/bootstrap_controller.ml

```diff
@@ -399,8 +402,8 @@ let run ~logger ~trust_system ~verifier ~network ~consensus_local_state
               | Error `Persistent_frontier_malformed ->
                   fail "persistent frontier was malformed"
             in
-            Logger.info logger ~module_:__MODULE__ ~location:__LOC__
-              "Bootstrap state: complete." ;
+            Logger.Structured.info logger ~module_:__MODULE__ ~location:__LOC__
```

Comment:
could use `Str` here

Reply:
Thanks. I was preferring to use `Structured` whenever there was enough space on the line that it wouldn't cause unnecessary extra wrapping; I can change this if it doesn't make sense to do this way.

Reply:
it's OK as-is